libs/ui/src/hooks/smartcard_auth/use_inserted_smartcard_auth.ts

```diff
@@ -140,7 +149,15 @@ function smartcardAuthReducer(allowedUserRoles: UserRole[], scope: AuthScope) {
                 scope
               );
               if (userResult.isOk()) {
-                return { status: 'logged_in', user: userResult.ok() };
+                const user = userResult.ok();
+                if (previousState.auth.status === 'logged_out') {
+                  // TODO either need to make this configurable or add passcode screen to BMD
```

Comment:
Can probably remove this TODO since you've prepared a PR (https://github.com/votingworks/vxsuite/pull/2032) for the latter (which I think was the right call since it's what we want from a product perspective)!